### DISCUSSION TITLE
enlarge the left panel in the admin ui

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#5118] [Ideas]
Hello,

I would like to share with you my frustration regarding your tree side bar.

When you work with large and complex pages, it is impossible to have access to a nested elements which is very very frustrated...

I know you are probably aware of this matter and I hope you can fix it soon :)

All the best

**Comment by shogunpurple** [comment]
@oswaldB this has been fixed and will be shipped in the next release. PR here https://github.com/Budibase/budibase/pull/4987

Before raising an issue, can you please make use of the search in our discussions forum to avoid raising duplicates.

A search for "side bar" in both issues and discussions yields results for this issue. 

Thanks
